bin/cylc-view

```diff
@@ -23,8 +23,7 @@ editor, after optional include-file inlining and Jinja2 preprocessing.
 
 The edit process is spawned in the foreground as follows:
   % <editor> suite.rc
-Where <editor> is defined in the cylc site and user config files
-($CYLC_DIR/conf/global.rc and $HOME/.cylc/global.rc).
+Where <editor> can be specific in cylc global config.
```

Comment:
Where <editor> can be ~specific~ *set* in cylc global config.

Reply:
oops, was prob meant to be 'specified'; but set is better...